week 5/README.md

```diff
@@ -109,6 +109,47 @@ for (int i = 0; i < n; ++i)
 
 ### Arrays
 
+The **array** is such a **data structure** that stores a fixed-size sequencial elements of a certain type.
+All elements of the array occupy contiguous memory locations. The lowest address corresponds to the first element and the highest address to the last element.
```

Comment:
I'm not sure that the address thingy is a useful explanation, we still haven't spoken about pointers